### PR TITLE
feat: workflow page added

### PR DESCRIPTION
**Pull request by Gabrielle-Barling**
I added the workflows page with vuetify. User can select a previous workflow (only default for now) or create a new workflow.

### CODE REVIEW

**Review by CadeTormanen — DISMISSED** [review]
lgtm!

**Review by amychisholm03 — CHANGES_REQUESTED** [review]
Please address the merge conflicts by using the updated sidebar with the `SimulationReports` routing.

File: vue-project/src/components/PrintJobs.vue [hunk 1]
```diff
@@ -149,47 +113,36 @@ const validateCreatePrintSettings = () => {
...
-    headers: { 'Content-Type': 'application/json' },
-    body: JSON.stringify(data)
-  });
+  //// API CALLS ////
+  const createPrintSettings = async () => {
+    if (!validateCreatePrintSettings()){
+      return false;
+    }   
+
+    const url = "http://54.200.253.84:80/createJob";
```
Comment (amychisholm03) [inline vue-project/src/components/PrintJobs.vue]: Might want to use `api.wsuv-hp-capstone.com` instead of IP, but we can change that later

**Review by amychisholm03 — DISMISSED** [review]

**Review by amychisholm03 — APPROVED** [review]
Saw this IRL looks good

**Review by CadeTormanen — APPROVED** [review]
lgtm

**Pull request merged**